### ISSUE TITLE
🐛 BUG: Cannot build outside/in parent of CWD

### ISSUE THREAD

**Issue by kaleidawave** [snowpackjs/astro#702]
### What package manager are you using?

yarn

### What operating system are you using?

windows

### Describe the Bug

Cannot build in parent of cwd. Build fails with `Error: Cannot delete files/directories outside the current working directory`

### Steps to Reproduce

With `astro.config.mjs`:
```js
export default {
  dist: '../../dist/',
```

Running `astro build`

```
[build] Error: Cannot delete files/directories outside the current working directory. Can be overridden with the `force` option.
    at safeCheck (node_modules\del\index.js:37:9)
    at mapper (node_modules\del\index.js:73:4)
    at node_modules\p-map\index.js:57:28
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by natemoo-re** [comment]
Thanks for opening an issue—I appreciate the clear reproduction steps _and_ the helpful error message from `del`!

I think I'm `+1` on setting `force: true` for `del`. If users are explicitly setting their `dist` dir outside of `cwd`, we should respect that.

**Comment by jasikpark** [comment]
What happens if I then set "dist: "/" 👀

**Comment by kaleidawave** [comment]
Thanks for the quick response :). Yep while I appreciate `del`s protection here, unfortunately due to some restrictions I need to build to a parent directory.

**Comment by kaleidawave** [comment]
Using `force` is a little risky and `"dist: "/"` could be very bad 💥 😆  Maybe could prevent building to absolute paths? or maybe if building outside of cwd and folder is non empty throw a error?

**Comment by FredKSchott** [comment]
Yea I'm -1 on adding force here without some other protection here. Maybe catch a thrown error and just turn this into a console warning? That way, the user could run the deletion themselves.

We may have adding some other protection here, or maybe inside of Snowpack? I'd also feel comfortable if we copied that protection code from Snowpack as well.

**Comment by jasikpark** [comment]
> We may have adding some other protection here, or maybe inside of Snowpack? I'd also feel comfortable if we copied that protection code from Snowpack as well.

If we can add that code, that sounds awesome! I can't find it though 😅

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
